NOTICE

```diff
@@ -26,3 +26,12 @@ Google's data interchange format, which can be obtained at:
     * license/LICENSE.protobuf.txt (New BSD License)
```

Comment:
yes. So, should we change it to LICENSE : Google Inc. or we can keep it as it is.